internal/bgp/frr/frr_test.go

```diff
@@ -31,9 +32,27 @@ func testOsHostname() (string, error) {
 
 func testCompareFiles(t *testing.T, configFile, goldenFile string) {
 	cmd := exec.Command("diff", configFile, goldenFile)
-	output, err := cmd.Output()
+	var lastError error
+
+	// Try comparing files multiple times because tests can generate more than one configuration
+	err := wait.PollImmediate(50*time.Millisecond, 2*time.Second, func() (bool, error) {
```

Comment:
Can we embed the check for the file presence here too?
Or, let's put the other way: with this, do we still need to wait for the file to be present?

Reply:
Good point! going to remove the file presence check, as it is embedded in the diff command